### DISCUSSION TITLE
BT-97-Lot mismatch definition ?

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#434] [Q&A] [answered]
Hi lab  !
As you wrote in doc : 

Tendering language (BT-97)
The languages that may be used to submit tenders (BT-97) shall be marked using [language](https://op.europa.eu/web/eu-vocabularies/at-dataset/-/resource/dataset/language) codes:

<cac:Language>
    <cbc:ID>ENG</cbc:ID>
</cac:Language>
<cac:Language>
    <cbc:ID>FRA</cbc:ID>
</cac:Language>

and that corresponds in xsd file.

So, this filed should not have attributes , instead in fields.json : 
{
****************
    "type" : "code",
    "legalType" : "CODE",
***********************
     } 
Bye
Renato

**Comment by YvesJo** [comment, marked as answer]
Correct, the XML element type for cbc:ID is Identifier and does not have a listName attribute. In JSON, one is referring to the type defined in the Regulation Annex and the other is there to specify that a list is used to restrict the possible values. While most of the time a listName exist, it is not always the case and besides having to look inside the XSD, there is currently no other available option. An alternate solution will be studied to help address that concern.
